materials/introduction/02-gem5-basics/02-frequently-used-features/02-class-variables.py

```diff
@@ -0,0 +1,69 @@
+class Processor:
+    type_name = "Processor"
+
+    def __init__(self, freq):
+        self.freq = freq
+
+    def change_type_name(self, new_name):
+        self.type_name = new_name  # this does not change Processor.type_name
+        # it creates an object variable instead
+
+
+class ProcessorWithClassFunction(Processor):
+    def __init__(self, freq):
+        super(ProcessorWithClassFunction, self).__init__(freq)
```

Comment:
I suggest using simply `super()` as it should work unless there's weird multiple inheritance.

Reply:
Done